react/tic-tac-toe/src/scss/_variables.scss

```diff
@@ -0,0 +1,4 @@
+$main-logo-size: 300px;
+$width-form-container: 444px;
+$m-top-30: -30px;
```

Comment:
This name is a bit confusing since it might mean `+30px on margin top` or `-30px on margn top`... Is there another way you can name it?